de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5552160_sys_gh6125UpdatedBankStatementDocType.sql

```diff
@@ -0,0 +1,10 @@
+update c_bankstatement
+set c_doctype_id = (select dt.c_docType_id 
+					from c_doctype dt 
+					where (dt.ad_org_id=c_bankstatement.ad_org_id or dt.ad_org_id = 0)
+					and dt.DocbaseType = 'CMB' 
+					and dt.DocSubType= 'BS' 
+					limit 1),
+	updatedBy = 99,
+	Updated=(select now())
```

Comment:
```suggestion
	Updated=('2020-02-14 09:15:32.812324+00') /*literal rsult of select now()*/
```

with
 
> the timestamp i just got from select now() the important thing is that you have one timestamp in the SQL

i meant to have the *literal* timestamp in the SQL. it's very crude, but that way we might have a chance of identifying the file via fulltext-search